Dockerfile.ci

```
@@ -330,8 +330,8 @@ RUN if [[ ${INSTALL_AIRFLOW_VIA_PIP} == "true" ]]; then \
 # they are also installed additionally to whatever is installed from Airflow.
 COPY docker-context-files/ /docker-context-files/
 
-RUN if [[ ${AIRFLOW_LOCAL_PIP_WHEELS} != "true" ]]; then \
-        if ls /docker-context-files/*.whl 1> /dev/null 2>&1; then \
+RUN if [[ ${INSTALL_FROM_DOCKER_CONTEXT_FILES} != "true" ]]; then \
+        if ls /docker-context-files/*.{whl,tar.gz} 1> /dev/null 2>&1; then \
             pip install --no-deps /docker-context-files/*.whl; \
```

```suggestion
            pip install --no-deps /docker-context-files/*.{whl,tar.gz}; \
```

You are listing for .tar.gz, but only installing .whl otherwise.

---

Very good point.